packages/wrangler/src/__tests__/dev.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { render } from "ink-testing-library";
+import patchConsole from "patch-console";
+import React from "react";
+import Dev from "../dev";
+import type { DevProps } from "../dev";
+
+describe("Dev component", () => {
+  let restoreConsole;
+  beforeEach(() => (restoreConsole = patchConsole(() => {})));
+  afterEach(() => restoreConsole());
```

Comment:
This is to stop the tests from looking really busy with extra logging.

Otherwise you see stuff like:

```
  console.error
    The above error occurred in the <Dev> component:
    
        at Dev (/Users/pbacondarwin/dev/wrangler2/packages/wrangler/src/dev.tsx:55:13)
        at ErrorBoundary (/Users/pbacondarwin/dev/wrangler2/node_modules/react-error-boundary/dist/react-error-boundary.cjs.js:59:35)
        at withErrorBoundary(Dev)
        at App (/Users/pbacondarwin/dev/wrangler2/node_modules/ink/build/components/App.js:42:9)
    
    React will try to recreate this component tree from scratch using the error boundary you provided, ErrorBoundary.
```

For some reason this doesn't seem to occur in real life.

Reply:
Having lots of error messages displayed when running tests is confusing to the person running the tests: are these supposed to be there (i.e. are they good errors??) or does it indicate that something is wrong in the tests?

I think you might be right that in normal process react (or Ink) is patching the console and sending these messages elsewhere - or... they are being displayed but then immediately hidden by the ErrorBoundary component.

Reply:
> I don't actually feel strongly about noisy consoles, BUT I'm concerned that you say it's not showing during actual usage. That's concerning. Maybe it has to do with how react hijacks console sometimes. I'mm investigate that later.

When I was doing the `autocompletion` work, I was seeing some error stack traces that looked very much like this, generally when I was doing something like entering `wrangler dev` with no filename path

Reply:
When this actually fails it is good to see these messages. But when the test is actually checking that an error is thrown, I don't believe we want to be displaying it in the console.